terraform/modules/rds_migration/ecs.tf

```diff
@@ -0,0 +1,128 @@
+#tfsec:ignore:aws-ecs-enable-container-insight:TODO CLDC-2542 enable container insights if necessary for logging/monitoring
+resource "aws_ecs_cluster" "this" {
+  #checkov:skip=CKV_AWS_65:TODO CLDC-2542 enable container insights if necessary for logging/monitoring
+  name = "${var.prefix}-db-migration"

```

Comment:
Do we want to rename the main ecs cluster to something like `${var.prefix}-app`?